components/post-header-info/post-header-info.tsx

```diff
@@ -0,0 +1,59 @@
+import Image from 'next/image';
```

Comment:
Is this component a duplicate of `post-header-data`?

Reply:
Yes, it wasn't supposed to be there